CoverMyDotNet/Client.cs

```diff
@@ -0,0 +1,68 @@
+﻿using RestSharp;
+using System.Net.Http;
+using System;
+
+namespace CoverMyDotNet
+{
+	public class Client : RestClient
+	{
+		private string _apiId;
+		private string _apiSecret;
+
+		public string ApiId { get { return _apiId; }}
+		public string ApiSecret { get { return _apiSecret; }}
+
+		public Client() : base()
+		{
+			this.BaseUrl = new Uri("https://api.covermymeds.com");
+			//try to get the api creds from the environment first
+			_apiId = Environment.GetEnvironmentVariable("cmm_api_id");
```

Comment:
Probably doesn't matter on Windows, but I think environment variables should typically be all-caps.